Frontend/src/app/views/dashboard/project-details/project-details.component.html

```diff
@@ -35,9 +35,19 @@ <h5>
             <input *ngIf="pageMode == 'edit'" type="text" status="primary" placeholder="Tags" nbTagInput (tagAdd)="onTagAdd($event)" fullWidth>
         </nb-tag-list>
     
-        <p>
+        <p *ngIf="pageMode == 'view'">
             <span>{{ details.description }}</span>
         </p>
+
+        <input 
+            *ngIf="pageMode == 'edit'" 
+            style="margin: 20px 0;"
```

Comment:
Probably don't want to be hunting down styles later on in the project and instead just keep everything in the scss

Reply:
fixed